lib/node_modules/@stdlib/math/base/special/log10/README.md

```diff
@@ -53,7 +53,7 @@ Evaluates the [common logarithm][common-logarithm].
 
 ```javascript
 var v = log10( 100.0 );
-// returns 2.0
+// returns ~2.0
```

Comment:
Why would this be approximate?

From the REPL,

```
In [1]: base.log10(100)
Out[1]: 2
```

Reply:
Apologies for it, I don't actually remember why I did this, but yeah as it is not approximate so we don't need ~.